ts/parts/DateTimeAxis.ts

```diff
@@ -211,6 +211,8 @@ class DateTimeAxis {
      */
     public static compose(AxisClass: typeof Axis): void {
 
+        AxisClass.keepProps.push('dateTime');
```

Comment:
So `dateTime` does need to be in `keepProps`? Why?

Reply:
For performance reasons, as we skip creating a new instance during re-init of the axis.

Reply:
Just to explain: It's similar to `logarithmic`, but the only action on `destroy` is to delete `axis.dateTime.axis`.

Reply:
> For performance reasons, as we skip creating a new instance during re-init of the axis.

Yes, that makes sense! 👍